projects/plugins/jetpack/extensions/blocks/dialogue/edit.js

```diff
@@ -178,38 +155,16 @@ export default function DialogueEdit( {
 
 				{ mediaSource && (
 					<MediaPlayerToolbarControl
-						onTimeChange={ ( time ) => setTimestamp( convertSecondsToTimeCode( time ) ) }
+						onTimeChange={ time => setTimestamp( convertSecondsToTimeCode( time ) ) }
 					/>
 				) }
-
-				{ currentParticipant && isFocusedOnParticipantLabel && (
```

Comment:
Do we need these buttons for styling the participant label? I'm not seeing them when I click:
<img width="274" alt="Screen Shot 2021-02-02 at 5 21 45 PM" src="https://user-images.githubusercontent.com/1689238/106670498-215c1e00-657b-11eb-8240-df50f7569e71.png">